src/cc_helpers.cc

```diff
@@ -1247,7 +1256,7 @@ bool certifier::framework::cc_trust_data::certify_secondary_domain(const string&
     }
   }
 
-  return found->certify_domain();
+  return (found ? found->certify_domain() : false);
```

Comment:
This fixes another seg-fault in this method. If supplied domain-name is not found, `found` will be NULL.

Handle that.